### ISSUE TITLE
Release 1.5

### ISSUE THREAD

**Issue by mpharrigan** [mdtraj/mdtraj#959]
Last release was June. There've been some major fixes and enhancement, my favourites include*

 - writing netcdf trajectories "correctly"
 - that pbc distances bug
 - rmsd with different reference / query indices

(* I don't have that fancy script that finds all the PRs merged since a release)

**Comment by swails** [comment]
+1

And I think you can go to the release page on github to see the list of commits since the last release.

**Comment by mpharrigan** [comment]
Yeah, but it includes all the commits which have helpful titles such as https://github.com/mdtraj/mdtraj/commit/4ca59d11464a0b1ac3210f523304ecc0b3c78aed :)

**Comment by rmcgibbo** [comment]
```
$ gh-util list-prs mdtraj/mdtraj
Listing merged PRs since: MDTraj 1.4.2

PR #955: Force clang on darwin
PR #954: [setup] require at least setuptools>=18 before doing anything.
PR #952: Use improved numpy histogram in RDF.
PR #951: Update BibTeX reference
PR #950: Pin version of msmb_theme
PR #945: Addresses mol2 reading bug #944
PR #943: [setup] do not call parent function via super, because Extension is an old-style class.
PR #942: Fix setup.py develop with latest version of setuptools
PR #935: Get the IPython notebook viewer working with IPython/Jupyter 4
PR #941: Update conda packaging
PR #939: Follow the Amber NetCDF Trajectory Spec (for compatibility with cpptraj)
PR #940: Update test_dssp.py
PR #938: Update dcd.pyx
PR #934: Automized setup (pip) to drag-in build time dependencies (numpy, cython)
PR #930: Fix MIC code for general triclinic unit cells.
PR #926: Fix dev upload
PR #925: Fix openmm-dev package uploads to binstar
PR #915: Superpose+rmsd with different reference, query indices
PR #922: Update trajectory_files.rst
PR #921: Update load_functions.rst
PR #917: Update analysis.rst
PR #911: Update README.md
PR #905: bugfix/dcdreader: fix sharing mode under Windows
PR #902: Update installation.rst
PR #901: Add  load_hoomdxml to top level __init__.py to supress sphinx docs warning.
PR #900: Hoomd fix
PR #899: MAINT: Use travis-ci container infastructure
PR #898: Add hash implementation
PR #897: Fix n_residues set by _topology_from_subset
PR #896: Fix #851, part 2
PR #885: Fix bug and sorting in hoomdxml reader
PR #893: add setup.py and basesetup.py as BSD?
PR #891: add BSD license to ./devtools folder and .travis.yml file
PR #889: typo in code example
PR #884: Spelling: principal component analysis
PR #862: ENH: More complete numpydoc coverage
PR #772: [iterload] added skip keyword to skip first n frames.
PR #871: One letter codes
PR #866: ENH: Fix GRO parser for files with higher precision.
PR #869: ENH: Some minor fixes suggested by QuantifiedCode.
PR #868: Update style.rst
PR #865: Added TrajectorySliderView
PR #861: Lammpstrj column ordering fixes
PR #860: Fix #858
```

**Comment by rmcgibbo** [comment]
+1 to release 1.5.0. is there anything we really need to fix first on the tracker?

**Comment by swails** [comment]
I couldn't see any at first glance, although we should start making better use of labels to make first glances more effective. As I can actually add them here, I will try to remember to do that.

**Comment by mpharrigan** [comment]
I closed all the old milestones, did a quick issue sweep and commented on things that might be relevant. I defer to others to create and tag issues with a 1.5 milestone

**Comment by mpharrigan** [comment]
I added milestone tags to what I think might be worth fixing before 1.5. If you think it can/should wait, switch the milestone to "1.6" or "wishlist"

**Comment by ajasja** [comment]
Yes, please a new release will be very welcome! (I hope this includes puting windows binaries on the omnia conda channel:)
<sub>
We are missing a very trivial function (toplogy.to_fasta). Right now we just copied this function into the 1.4.2, but this is a very hackish solution. 
</sub>

**Comment by cxhernandez** [comment]
I think once @rafwiewiora resolves https://github.com/mdtraj/mdtraj/issues/977, we'll be all set to release 1.5.

**Comment by mpharrigan** [comment]
Let's just do it now. I'll do a final update of the changelog and start doing the release

**Comment by mpharrigan** [comment]
uploaded to pypi (users please test `pip install mdtraj`)

Waiting for merge on conda-recipes for conda packages to build

**Comment by cxhernandez** [comment]
:+1:

**Comment by mpharrigan** [comment]
bug in dihedral changes

**Comment by rafwiewiora** [comment]
#977 resolved in #980 if you also want to incorporate that now.
